sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -62,3 +64,29 @@ export interface StudentRestrictionDTO {
   hasProvincialRestriction: boolean;
   restrictionMessage: string;
 }
+/**
+ * DTO object for student details.
+ */
+export interface StudentDetailDTO {
+  firstName: string;
+  lastName: string;
+  email: string;
+  gender: string;
+  dateOfBirth: Date;
+  contact: StudentContact;
+  pdVerified: boolean;
+  pdStatus: StudentPDStatus;
+  applications: StudentApplicationSummary[];
```

Comment:
I would recommend creating two separate API calls for this. They belong to two separated contexts, that, right now, are being displayed on the same UI but, from an API perspective, it does not matter much. Does it make sense?
Assume that we would be adding new features to the list, like sorting, pagination, etc, that will need new API calls. Why we would be retrieving Student+Applications all the time?